components/eam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -675,127 +679,98 @@ subroutine aero_model_wetdep(dt, dlf, dlf2, cmfmc2, state,                    &!
-                                                ! *** AND ARE GATHERED ***
-    real(r8),            intent(in)    :: mu(pcols,pver)   ! Updraft mass flux (positive)
-    real(r8),            intent(in)    :: md(pcols,pver)   ! Downdraft mass flux (negative)
-    real(r8),            intent(in)    :: du(pcols,pver)   ! Mass detrain rate from updraft
-    real(r8),            intent(in)    :: eu(pcols,pver)   ! Mass entrain rate into updraft
-    real(r8),            intent(in)    :: ed(pcols,pver)   ! Mass entrain rate into downdraft
+    real(r8),            intent(in)    :: dlf2(:,:)   ! Shal conv cldwtr detrainment [kg/kg/s]
+    real(r8),            intent(in)    :: cmfmc2(pcols,pverp) ! Shal conv mass flux [kg/m2/s]
+    real(r8),            intent(in)    :: sh_e_ed_ratio(pcols,pver)  ! shallow conv [ent/(ent+det)] ratio [fraction]
+    ! mu, md, ..., ideep, lengath are all deep conv variables
+    ! *** AND ARE GATHERED ***
     ! eu, ed, du are "d(massflux)/dp" and are all positive
-    real(r8),            intent(in)    :: dp(pcols,pver)   ! Delta pressure between interfaces
-    real(r8),            intent(in)    :: dsubcld(pcols)   ! Delta pressure from cloud base to sfc
+    real(r8),            intent(in)    :: mu(pcols,pver)   ! Updraft mass flux (positive) [mb/s]
+    real(r8),            intent(in)    :: md(pcols,pver)   ! Downdraft mass flux (negative) [mb/s]
+    real(r8),            intent(in)    :: du(pcols,pver)   ! Mass detrain rate from updraft [1/s]
+    real(r8),            intent(in)    :: eu(pcols,pver)   ! Mass entrain rate into updraft [1/s]
+    real(r8),            intent(in)    :: ed(pcols,pver)   ! Mass entrain rate into downdraft [1/s]
+    real(r8),            intent(in)    :: dp(pcols,pver)   ! Delta pressure between interfaces [mb]
     
     integer,             intent(in)    :: jt(pcols)         ! Index of cloud top for each column
     integer,             intent(in)    :: maxg(pcols)       ! Index of cloud top for each column
     integer,             intent(in)    :: ideep(pcols)      ! Gathering array
     integer,             intent(in)    :: lengath           ! Gathered min lon indices over which to operate
-    integer,             intent(in)    :: species_class(:)
+    integer,             intent(in)    :: species_class(:)  ! species index
     
-    type(cam_out_t),     intent(inout) :: cam_out     ! export state
-    type(physics_buffer_desc), pointer :: pbuf(:)
-
-    type(physics_ptend), intent(out)   :: ptend       ! indivdual parameterization tendencies
-
-
 
     ! local vars
+    type(wetdep_inputs_t) :: dep_inputs ! wet deposition variables defined from wetdep_inputs_set
 
-    integer :: i
-    integer :: jnv ! index for scavcoefnv 3rd dimension
+    integer :: jnv         ! index for scavcoefnv 3rd dimension
     integer :: jnummaswtr  ! indicates current aerosol species type (0 = number, 1 = dry mass, 2 = water)
-    integer, parameter :: jaeronumb=0, jaeromass=1, jaerowater=2
```

Comment:
To me, it's hard to say whether we should use 0, 1, and 2 to replace jaeronumb, jaeromass, jaerowater.

Reply:
I also think it is better to use `jaeronumb, jaeromass, jaerowater` or more explicit names that are declared as all caps parameters. It makes the code easy to read.

Reply:
added them back